default/AI/ShipDesignAI.py

```diff
@@ -1445,7 +1404,7 @@ def _rating_function(self):
         if self.troops == 0:
             return INVALID_DESIGN_RATING
         else:
-            return self.troops/(self.production_cost**((1+foAI.foAIstate.shipCount * AIDependencies.SHIP_UPKEEP)**-1))
+            return self.troops/self._adjusted_production_cost()
```

Comment:
> It is all about spaces. :)
```python
return self.troops / self._adjusted_production_cost()
```

just to be clear, this change you now propose there is purely a matter of preference, not covered by the guideline that was the entire body of your first comment, right?